bin/rbenv-ctags

```diff
@@ -17,6 +17,16 @@ generate_ctags_in() {
   (cd "$1"; ctags -R --languages=ruby)
 }
 
+generate_c_ctags_in() {
+  if [ -n "$RUBY_BUILD_BUILD_PATH" ]; then
+    echo "Running ctags on $RUBY_BUILD_BUILD_PATH"
```

Comment:
I am generally in agreement, but this is only doing what I already do elsewhere. I think my rationale was that the compilation output was already pretty noisy, and this is (debatably) useful information.